cmd/control-router.go

```diff
@@ -79,7 +79,7 @@ type controlAPIHandlers struct {
 }
 
 // Register control RPC handlers.
-func registerControlRPCRouter(mux *router.Router, srvCmdConfig serverCmdConfig) {
+func registerControlRPCRouter(mux *router.Router, srvCmdConfig serverCmdConfig) (err error) {
```

Comment:
@bobinette nice catch. It doesn't serve its intended purpose in this function (and in ``registerDistNSLockRouter``). I use the named return approach in ``configureServerHandler`` function, where ``err`` is assigned possibly non-nil values inside an if-block, to avoid declaring ``var err error`` outside of the if-block.
I shall address this comment in my next update to this PR.